### DISCUSSION TITLE
Why is the last path still colored after the order is completed

### DISCUSSION THREAD

**Discussion by HuJunnj** [openTCS/opentcs#124] [General]
Why is the last path still colored after the order is finished?
<img width="710" alt="Screenshot 2024-01-29 225104" src="https://github.com/openTCS/opentcs/assets/75943306/efda222a-c158-4467-aff7-28ca92003a9a">

**Comment by swltr** [comment]
@HuJunnj Because the vehicle is still occupying the path behind it as a resource. The vehicle has a length, and it reaches back into that path.

**Reply by vip404** [reply]
but if count length ,Point-0002 to Point-0003 is occupying too (i know it need to request occupy ,but actually occupying)?

**Reply by swltr** [reply]
If the vehicle hasn't been sent to Point-0003, yet, why should it occupy the path leading to it?
